app/grandchallenge/reader_studies/tasks.py

```diff
@@ -37,3 +44,11 @@ def add_images_to_reader_study(*, upload_session_pk, reader_study_pk):
     reader_study = ReaderStudy.objects.get(pk=reader_study_pk)
 
     reader_study.images.add(*images.all())
+
+
+@shared_task
+def add_image_to_answer(*, upload_session_pk, answer_pk):
+    image = Image.objects.filter(origin_id=upload_session_pk).first()
```

Comment:
I think it would be good to error out if more than 1 image is uploaded, that would be something for the workstation/client developers to fix. What do you think?